shared/util.js

```diff
@@ -270,7 +280,10 @@ class Util {
     return {
       file: Util.getURLDisplayName(parsedUrl),
       hostname: parsedUrl.hostname,
-      origin: parsedUrl.origin,
+      // Node's URL parsing behavior is different than Chrome and returns 'null'
+      // for chrome-extension:// URLs. See https://github.com/nodejs/node/issues/21955.
+      origin: parsedUrl.protocol === 'chrome-extension:' ?
```

Comment:
What if we used `UrlUtils.getOrigin` here instead so we aren't duping this behavior?